application/cloud_api.py

```diff
@@ -621,6 +621,51 @@ def speedDistribution(self, identifier, file_path):
         return (True, None)
 
 ###############################################################################
+# Compare Methods
+###############################################################################
+
+    def compareSpeeds(self, identifier, identifiers_to_cmp, labels_to_cmp, file_path, only_show_85th=False):
+        print "compareSpeeds called with identifer = {},\n identifiers_to_cmp = {},\n, labels_to_cmp = {},\n only_show_85th = {}"\
+                .format(identifier, identifiers_to_cmp, labels_to_cmp, only_show_85th)
+
+        payload = {
+            'identifier': identifier,
+            'identifiers_to_cmp': identifiers_to_cmp,
+            'labels_to_cmp': labels_to_cmp,
+            'only_show_85th': only_show_85th
+        }
+
+        try:
+            r = requests.get(self.server_addr + 'compareSpeeds', params = payload, stream=True)
+        except requests.exceptions.ConnectionError as e:
+            print('Connection is offline')
+            return (False, 'Connection to server "{}" is offline'.format(self.server_addr))
```

Comment:
Due to changes made in #102 , this will always need to be a 3-tuple. Once that is merged, your PR needs to be updated to reflect this.